src/vs/workbench/contrib/url/browser/trustedDomainsValidator.ts

```diff
@@ -103,56 +106,56 @@ export class OpenerValidatorContributions implements IWorkbenchContribution {
 				formattedLink += linkTail.charAt(0) + '...' + linkTail.substring(linkTail.length - linkTailLengthToKeep + 1);
 			}
 
-			const { choice } = await this._dialogService.show(
-				Severity.Info,
-				localize(
+			const { result } = await this._dialogService.prompt<boolean>({
```

Comment:
@sbatten fyi please check, this one is a bit more complex adoption of the new dialog api